src/components/top-page/top-page-content.styles.scss

```diff
@@ -0,0 +1,38 @@
+.japanese-text {
+    color: white;
+    font-family: YuMincho;
+    font-size: 3vw;
```

Comment:
this is tricky. usually vw/vh isn't used for text. it doesn't allow users to zoom and also you cant control going below an unreadable font size (it is common practice to stay at 16px+). the english text is more for show and might be alright, but i reckon either px or em is better for the Japanese text. up to you though.

Reply:
this just popped up on my recommendations... I'm being watched 🧐
Might help with choosing what measurements to use in what circumstance 
https://www.youtube.com/watch?v=N5wpD9Ov_To